### ISSUE TITLE
ClassCastException when I call the download function

### ISSUE THREAD

**Issue by adinlead** [wtekiela/opensub4j#11]
```
java.lang.ClassCastException: java.lang.Boolean cannot be cast to [Ljava.lang.Object;
	at com.github.wtekiela.opensub4j.impl.ResponseParser.bindList(ResponseParser.java:28)
	at com.github.wtekiela.opensub4j.impl.ListOperation.execute(ListOperation.java:28)
	at com.github.wtekiela.opensub4j.impl.OpenSubtitlesClientImpl.downloadSubtitles(OpenSubtitlesClientImpl.java:147)
	at Test.main(Test.java:20)
```

**Comment by adinlead** [comment]
```
public static void main(String[] args) throws XmlRpcException {
        OpenSubtitlesClient openSubtitlesClient = Main.makeOpenSubtitlesClient();
        if (openSubtitlesClient != null && openSubtitlesClient.isLoggedIn()) {
            List<SubtitleInfo> subtitleInfoList = openSubtitlesClient.searchSubtitles("eng", "friends", "1", "1");
            if (subtitleInfoList.size() > 0) {
                System.out.println(subtitleInfoList.size());
                for (SubtitleInfo subtitleInfo : subtitleInfoList) {
                    try {
                        System.out.println(subtitleInfo.toString());
                        List<SubtitleFile> subtitleFiles = openSubtitlesClient.downloadSubtitles(subtitleInfo.getId());
                    } catch (ClassCastException e) {
                        e.printStackTrace();
                    }
                }
            }
            openSubtitlesClient.logout();
        }
    }
```

**Comment by wtekiela** [comment]
Hi @adinlead, you pass wrong ID to download subtitles function. Of course error handling needs to be improved as stated in #10.

If you call
```
openSubtitlesClient.downloadSubtitles(subtitleInfo.getSubtitleFileId());
```
instead of 
```
openSubtitlesClient.downloadSubtitles(subtitleInfo.getId());
```
your code works fine.

**Comment by wtekiela** [comment]
Fixed error handling in #16

**Comment by wtekiela** [comment]
Released in 0.2.3
